packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/http_response_error.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// Error returned in `PlatformNavigationDelegate.setOnHttpError` when an HTTP
+/// response error has been received.
+/// Platform specific implementations can add additional fields by extending
+/// this class.
+///
+/// {@tool sample}
```

Comment:
You can remove this line since `pub.dev` doesn't support it. I know this shows up in the other classes, but I haven't had the chance to update them yet.